### DISCUSSION TITLE
There are too many DOM elements in el-drawer, so it is very slow when opening el-drawer for the first time.

### DISCUSSION THREAD

**Discussion by lyqyq2001** [element-plus/element-plus#17595] [Q&A]
How to optimize it?

**Comment by warmthsea** [comment]
> How to optimize it?

1、Add `v-loading` animation to the button or page that opens the pop-up window.
2、The elements in `el-drawer` are rendered after opening, and loading animation can be added during rendering
3、Perf [code](https://github.com/element-plus/element-plus/tree/dev/packages/components/drawer)
